src/bootstrap/test.rs

```diff
@@ -846,12 +846,30 @@ fn check_if_browser_ui_test_is_installed_global(npm: &Path, global: bool) -> boo
         .output()
         .map(|output| String::from_utf8_lossy(&output.stdout).into_owned())
         .unwrap_or(String::new());
-    lines.contains(&" browser-ui-test@")
+    lines
+        .lines()
+        .filter_map(|l| l.split(" browser-ui-test@").skip(1).next().map(|v| v.to_owned()))
```

Comment:
It looks like npm list has a --parseable flag, and in combination with --long that might be better for us to use -- we could then find_map and strip a `.*:browser-ui-test@` prefix, which seems more reliable than the current setup.

Reply:
Thanks a lot for all this new information! Will make this much simpler.